lib/hanami/logger.rb

```diff
@@ -91,8 +92,33 @@ class Formatter < ::Logger::Formatter
       #
       # @see http://www.ruby-doc.org/stdlib/libdoc/logger/rdoc/Logger/Formatter.html#method-i-call
       def call(severity, time, progname, msg)
-        progname = "[#{@application_name}] #{progname}"
-        super(severity, time.utc, progname, msg)
+        hash = {
+          application_name: @application_name,
+          severity: severity,
+          time: time.utc,
+          progname: progname
```

Comment:
@davydovanton Is `progname` still used and/or useful? It's always null in the tests.